RCAIDE/Library/Methods/Thermal_Management/Reservoirs/Reservoir_Tank/compute_mixing_temperature.py

```diff
@@ -0,0 +1,81 @@
+# RRCAIDE/Library/Methods/Thermal_Management/Reservoirs/Reservoir_Tank/compute_mixing_temperature.py
+
+
+# Created:  Apr 2024, S. Shekar 
+
+# ---------------------------------------------------------------------------------------------------------------------- 
+#  IMPORT
+# ---------------------------------------------------------------------------------------------------------------------- 
+import  RCAIDE
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  Compute resultant temperature of the reservoir
+# ----------------------------------------------------------------------------------------------------------------------
+def compute_mixing_temperature(RES,state,coolant_line,dt,i):
+    
```

Comment:
Remove blank space and improve function header inputs. like the average user does not know what is "HAS"

Reply:
Done